<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>629</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DB-GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="ErrorLable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QListView" name="TablesList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="batchSize">
         <number>100</number>
        </property>
       </widget>
       <widget class="QTableView" name="CurrentTable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="History">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPlainTextEdit" name="CommandTextEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionClearEdit"/>
    <addaction name="actionClearHistory"/>
   </widget>
   <widget class="QMenu" name="menuMeetings">
    <property name="title">
     <string>Meetings</string>
    </property>
    <addaction name="actionCreateMenTable"/>
    <addaction name="actionDropMenTable"/>
    <addaction name="separator"/>
    <addaction name="actionCreateWomenTable"/>
    <addaction name="actionDropWomenTable"/>
    <addaction name="separator"/>
    <addaction name="actionCreateMeetingsTable"/>
    <addaction name="actionDropMeetingsTable"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="actionIndexSearchBTREE"/>
    <addaction name="actionIndexSearchHash"/>
    <addaction name="separator"/>
    <addaction name="actionAnalyzeSearchBTree"/>
    <addaction name="actionAnalyzeIndexSearchBTREE"/>
    <addaction name="actionAnalyzeIndexSearchHash"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuMeetings"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClearEdit">
   <property name="text">
    <string>Clear edit</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="text">
    <string>Clear history</string>
   </property>
  </action>
  <action name="actionDropMenTable">
   <property name="text">
    <string>Drop men table</string>
   </property>
  </action>
  <action name="actionDropWomenTable">
   <property name="text">
    <string>Drop women table</string>
   </property>
  </action>
  <action name="actionCreateMeetingsTable">
   <property name="text">
    <string>Create meetings table</string>
   </property>
  </action>
  <action name="actionDropMeetingsTable">
   <property name="text">
    <string>Drop meetings table</string>
   </property>
  </action>
  <action name="actionCreateWomenTable">
   <property name="text">
    <string>Create women table</string>
   </property>
  </action>
  <action name="actionCreateMenTable">
   <property name="text">
    <string>Create men table</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionIndexSearchBTREE">
   <property name="text">
    <string>Search with indexes BTREE</string>
   </property>
  </action>
  <action name="actionAnalyzeSearchBTree">
   <property name="text">
    <string>Analyze search</string>
   </property>
  </action>
  <action name="actionAnalyzeIndexSearchBTREE">
   <property name="text">
    <string>Analyze index search BTREE</string>
   </property>
  </action>
  <action name="actionIndexSearchHash">
   <property name="text">
    <string>Search with indexes Hash</string>
   </property>
  </action>
  <action name="actionAnalyzeIndexSearchHash">
   <property name="text">
    <string>Analyze index search Hash</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
